docs/frontend.md

```diff
@@ -6,10 +6,10 @@ description: This document describes approaches that you can use to implement a
 
```

Comment:
Please change line 4 above:

`description: This document describes approaches that you can use to implement a frontend for a reSolve application.`

Please change "approaches" to "techniques".

Reply:
Done